src/bindings/python/fluxacct/accounting/job_archive_interface.py

```diff
@@ -304,7 +336,7 @@ def apply_decay_factor(decay, acct_conn, user=None, bank=None):
         acct_conn.commit()
         period += 1
 
-    return sum(usg_past_decay)
+    return sum(usg_past_decay[:-1])
```

Comment:
Can you add a comment here to make clear why the `[-1]` is being used (I think your commit message makes it clear, but it'll be good to make it clear here too).

Reply:
Sure thing. I pushed a `--fixup` commit that adds a comment just above the `return` statement that explains why we don't return the last factor.